account_asset_management_number/views/account_asset.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="account_asset_view_form">
+        <field name="name">account.asset.form</field>
+        <field name="model">account.asset</field>
+        <field
+            name="inherit_id"
+            ref="account_asset_management.account_asset_view_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/group/field[@name='code']" position="before">
+                <field name="number_by_sequence" invisible="True" />
+                <field
+                    name="number"
+                    attrs="{'readonly':[('number_by_sequence', '=', True)]}"
+                />
```

Comment:
![Selection_145](https://user-images.githubusercontent.com/1973598/133579000-14bae4f1-a9fc-48d8-8e42-51ae20d1bb71.png)
```suggestion
            <h1 position="after">
                <field name="number_by_sequence" invisible="True" />
                <div attrs="{'readonly':[('number_by_sequence', '=', True)]}">
                    <label for="number" class="oe_edit_only" />
                    <h1>
                        <field name="number" class="oe_inline" />
                    </h1>
                </div>
            </h1>
```

Reply:
I think using `<h2>`  instead of `<h1>` would be better.

![Selection_154](https://user-images.githubusercontent.com/51266019/133664970-9d15894b-e039-40ca-92de-f00c92dc3bda.png)